### PR TITLE
DOC: added some doc cells about what's going on

### PR DESCRIPTION
**Pull request by wasade**
Should be a quick review

### CODE REVIEW

File: ipynb/Enterotypes.ipynb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we're going to explore the relationship between enterotype taxa and the American Gut fecal samples."
+   ]
```
Comment (jwdebelius) [inline ipynb/Enterotypes.ipynb]: Do we need to add a citation to the paper, or some other paper to justify the comparison?

File: ipynb/Enterotypes.ipynb [hunk 2]
```diff
@@ -82,6 +116,13 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And plot"
```
Comment (jwdebelius) [inline ipynb/Enterotypes.ipynb]: Could this be a complete sentence?

**Comment by jwdebelius** [comment]
A few minor comments.

Thanks!

**Comment by wasade** [comment]
I don't see any comments...?

**Review by mortonjt — APPROVED** [review]

**Pull request merged**